docs/blog/_posts/2019-12-20-21th-dotty-milestone-release.md

```diff
@@ -0,0 +1,329 @@
+---
+layout: blog-page
+title: Announcing Dotty 0.21.0-RC1 – ❄️ Feature Freeze ❄️
```

Comment:
Drop – ❄️ Feature Freeze ❄️from the title; since it could be misleading. We are really talking about being "feature complete" in the sense that everything we envisioned for Scala 3 is now in the release.

But individual features can still change, or be delayed, or be dropped depending on experience gained, and feedback we receive from the community and the SIP committee.